### DISCUSSION TITLE
rigid-rigid contact

### DISCUSSION THREAD

**Discussion by HNUZhaoXiang** [InteractiveComputerGraphics/SPlisHSPlasH#234] [Q&A]
hi
How do splishsplash implement the rigid-rigid coupling calculation?

**Comment by janbender** [comment]
Rigid-rigid coupling is implemented using the [PBD](https://github.com/InteractiveComputerGraphics/PositionBasedDynamics) library. This also enables to link rigid bodies by joints or motors.
